.travis.yml

```diff
@@ -7,7 +7,7 @@ before_script:
 - "sudo docker run -d --name aerospike -p 3000:3000 -p 3001:3001 -p 3002:3002 -p 3003:3003 aerospike"
 after_script:
   - bash -ex test/coveralls.sh
-script: lein do clean, javac, test
+script: lein do clean, test, test :integration
```

Comment:
Why not run all the tests together?

Reply:
Yeah, but you are not separating them into two jobs.
It's still a single job, and you will have to see the errors in the logs.
I don't really get what separation you are referring to.

Reply:
OK, changed to `test :all`.